assets/src/blocks/CarouselHeader/SidebarSlide.js

```diff
@@ -23,45 +36,104 @@ export const SidebarSlide = ({
     changeSlideAttribute('focal_points', index)(focalPoints);
   }
 
+  const onToggleHandler = (opened) => {
+    setIsOpened(opened);
+  }
+
+  useEffect(() => {
+    if(isOpened) {
+      goToSlideHandler(index);
+    }
+  }, [ isOpened ]);
+
   return (
-    <>
-      <div className='carousel-header-slide-container'>
-        <div className='carousel-header-slide-options-wrapper'>
-          <div className='row'>
-            <div className='col'>
-              <URLInput
-                label={__('Url for link', 'planet4-blocks-backend')}
-                value={link_url}
-                onChange={changeSlideAttribute('link_url', index)}
-              />
+    <div
+      className={`sidebar-slide ${isOpened ? 'opened-slide' : ''}`}
+      data-index={index}
+      draggable={!isOpened}
+      onDragStart={onDragStartHandler}
+      onDragEnd={onDragEndHandler}
+      onDragOver={onDragOverHandler}
+      onDragLeave={onDragLeaveHandler}
+    >
+      <PanelBody
+        key={ index }
+        initialOpen={ isOpened }
+        onToggle={onToggleHandler}
+        title={
+          <div className='slide-item-wrapper'>
+            <div
+              className='slide-item-draggable-button'
+              onClick={(evt) => {
+                // This method avoids to propagate the toggle event
+                evt.stopPropagation();
+              }}
+            >
+              <svg width="8" height="14" viewBox="0 0 8 14" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="2" height="2" fill="black"/><rect x="6" width="2" height="2" fill="black"/><rect y="6" width="2" height="2" fill="black"/><rect x="6" y="6" width="2" height="2" fill="black"/><rect y="12" width="2" height="2" fill="black"/><rect x="6" y="12" width="2" height="2" fill="black"/></svg>
             </div>
-          </div>
-          <div className='row'>
-            <div className='col'>
-              <CheckboxControl
-                label={__('Open in a new tab', 'planet4-blocks-backend')}
-                value={link_url_new_tab}
-                checked={link_url_new_tab}
-                onChange={changeSlideAttribute('link_url_new_tab', index)}
-              />
+            <div className='slide-item-order-arrows'>
+              <div
+                className={`arrow-button ${index === 0 ? 'disabled' : ''}`}
+                onClick={ upOrDownHandler }
+                data-index={index}
+                data-type='up' />
+              <div
+                className={`arrow-button ${isLastItem ? 'disabled' : ''}`}
+                onClick={ upOrDownHandler }
+                data-index={index}
+                data-type='down' />
             </div>
+            <img
+              className='slide-item-image'
+              draggable={ false }
+              srcSet={ image_srcset }
```

Comment:
If we also add `sizes='50px'` here it will help the browser load the right size.